manifests/k/KDE/rkward/0.7.2/KDE.rkward.yaml

```diff
@@ -1,4 +1,4 @@
-PackageIdentifier: kde.rkward
+PackageIdentifier: KDE.rkward
 Publisher: kde
 PackageName: rkward
```

Comment:
Uhh the PackageVersion is `master-1cde409a`
![image](https://user-images.githubusercontent.com/15158490/129444542-bb2fa766-a70e-41ae-98fe-784e68fb944e.png)

```suggestion
Publisher: KDE e.V.
PackageName: RKWard
```